README.md

```diff
@@ -20,4 +20,32 @@ This repository is configured for Nat Pryce's [_adr-tools_](https://github.com/n
 
 API definitions are in OAS (Swagger 2.0).
 
-# The Digital Citizenship Initiative
+## Deploy Azure PaaS
+
+This repository contains scripts to deploy needed PaaS on Azure cloud.
+
+### Prerequisites
+
+- [Git](https://git-scm.com/)
+- [NodeJS](https://nodejs.org/it/)
+- [Terraform](https://terraform.io) 
+
+All binaries must be in the system path.
+
+### Deploy instructions
+
+- Get an [Azure account](https://azure.microsoft.com/en-us/free)
+- Set up an [Active Directory Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-application-objects)
+- Set up environment variables:
+
+```
+export ARM_SUBSCRIPTION_ID=<subscription Id>
+export ARM_CLIENT_ID=<service principal client (app) Id>
+export ARM_CLIENT_SECRET=<service principal client secret (key)>
+export ARM_TENANT_ID=<Active Directory domain Id>
+```
+
+- Run the following commands:
```

Comment:
explain what is going to happen by running these commands

Reply:
+1